Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "efc679ed3f43fa36d6a77a7d8c2304667e363906ebb9209804504700db8439b2"
+            "sha256": "83dc036e85512d71c360df5dff768f700a20cca0be5bef5cac75a24c882506a7"
```

Comment:
I think we're syncing this file to the local container rather than copying it and that's why it's changing all the time.  Everytime the container is rebuilt it does this.   Fine for now, but we should fix this at some point.